proxy/grpcproxy/metrics.go

```diff
@@ -29,12 +29,6 @@ var (
 		Name:      "events_coalescing_total",
 		Help:      "Total number of events coalescing",
 	})
-	cacheKeys = prometheus.NewGauge(prometheus.GaugeOpts{
```

Comment:
Since there's no way to get the number of keys, this metric can't be computed efficiently.